Makefile

```diff
@@ -1,4 +1,4 @@
-CFLAGS+=	-Wall -Wno-parentheses -Wno-missing-braces -O3 -g -DNDEBUG
+CFLAGS+=	-Wall -Wno-parentheses -Wno-missing-braces -O3 -g -DNDEBUG -march=native
```

Comment:
I omitted `-march=native` as it's only available on x86.  This makefile is just for research and testing purposes, you are encouraged to supply custom CFLAGS through the environment.